### ISSUE TITLE
Two `svg` elements, using VictoryLine

### ISSUE THREAD

**Issue by BrodaNoel** [FormidableLabs/victory#611]
Guys.
This is my code:

```
              <VictoryChart theme={VictoryTheme.material}>
                <VictoryLine
                  data={...}
                  theme={VictoryTheme.material} />

                <VictoryLine
                  data={...}
                  theme={VictoryTheme.material} />

                <VictoryLegend
                  data={[{ name: 'Retweets' }, { name: 'Favorites' }]} />
              </VictoryChart>
```

And this is what I'm getting:
![screen shot 2017-06-05 at 2 46 47 pm](https://cloud.githubusercontent.com/assets/1837969/26795942/ebb46bc8-49fd-11e7-890a-b1d58b4dae46.png)

Is it ok?

**Comment by chrisbolin** [comment]
hey @BrodaNoel! it sure is OK. It's used so that certain elements can overflow if necessary. I believe it was added in `victory-chart@19.1.0`, which was included in `victory@0.19.1`

**Comment by tobiaslins** [comment]
Is it possible to use it without a second svg?
With this it is not possible to just render the svg on the server and return it as an image. 
It is not an valid svg then
